packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.ts

```diff
@@ -164,6 +204,27 @@ export function interactiveGraphReducer<
                     }),
                 }),
             };
+        }
+        default:
+            return state;
+    }
+}
+
+// Generic type makes returned state match input state
+export function interactiveGraphReducer(
+    state: InteractiveGraphState,
+    action: InteractiveGraphAction,
+): InteractiveGraphState {
```

Comment:
Since this `interactiveGraphReducer` is the main export of this file, IMO it should be at the top of the file where people can easily find it.